tests/out/glsl/push-constants.main.Fragment.glsl

```diff
@@ -9,7 +9,7 @@ struct PushConstants {
 struct FragmentIn {
     vec4 color;
 };
-layout(std140, binding = 4) uniform PushConstants_block_0Fragment { PushConstants pc; };
+uniform PushConstants pc;
```

Comment:
does that work? will it show up as separate constants for the fields?

Reply:
Yes OpenGL will report each field of the struct separately